src/Apps/Collect/Routes/Collection/Components/Header/index.tsx

```diff
@@ -28,21 +28,21 @@ export const CollectionHeader: React.FC<CollectionHeaderProps> = ({
   artworks,
   collection,
 }) => {
-  const hasMultipleArtists =
```

Comment:
At first glance this looks to be already guarded in this file, but also.... /shrug

(btw the Sentry error - https://artsynet.sentry.io/issues/2343582424/?project=28316&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=24h&stream_index=13 - doesn't have a good source-map, makes it tougher to pinpoint)